### PR TITLE
fix cancel order button

### LINKED ISSUE #11341: Cancel order dropdown not working on tabs other than order details
## Description

The dropdown Action > Cancel order on edit order page is not working apart from the one on order details tab

## Steps to Reproduce

1. Go to `admin/orders`
2. Edit an order
3. See cancel button is not working on customer, payment and adjustment tab

## Workaround
Use order details page

## Severity
bug-s3: a feature is broken but there is a workaround

### PR DESCRIPTION
**Pull request by abdellani**
#### What? Why?

- Closes #11341

The confirmation model was not popping up after clicking on the `cancel order` button because the partial that includes the component (`spree/admin/shared/custom-confirm`) was not included on the order pages: adjustments, payments etc...
It was working only  on the edit order page because one of the nested partials includes that component 

```
app/views/spree/admin/orders/edit.html.haml
-> app/views/spree/admin/orders/_form.html.haml
--> app/views/spree/admin/orders/_shipment.html.haml
---> app/views/spree/admin/orders/_shipment_manifest.html.haml
---->  spree/admin/shared/custom-confirm
```
image

#### What should we test?

- try to cancel an order from every page under order details
image

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Review by mkllnk — APPROVED** [review]

File: app/views/spree/admin/shared/_order_links.html.haml [hunk 1]
```diff
@@ -10,3 +10,5 @@
           %span
             %i{ class: link[:icon] }
           %span=link[:name]
+
+= render 'spree/admin/shared/custom-confirm'
```
Comment (jibees) [inline app/views/spree/admin/shared/_order_links.html.haml:14]:
```suggestion
= render 'spree/admin/shared/custom-confirm'

```

**Review by jibees — APPROVED** [review]
Nice find, thanks for the specs! 🙏 

One small suggestion.

**Review by jibees — APPROVED** [review]
I'd squash the two commits but 👍

**Comment by drummer83** [comment]
Hi @abdellani,
Thanks for this one!

### Before staging
- I verified that cancelling orders is only working on the page 'Order Details'. :heavy_check_mark: 

### After staging
- Cancelling orders via dropdown menu is now working on all tabs of an order. :heavy_check_mark: 
- It's also working on the (still to come) invoices tab. :heavy_check_mark: 
- Other actions from the menu are still working. :heavy_check_mark: 
- Cancelling an order via the 'x Cancel' button is still working. :heavy_check_mark: 
- Cancelling orders in bulk is still working. :heavy_check_mark: 
- The above is working for super admin users as well. :heavy_check_mark: 

### Conclusion
I couldn't find any issues with this. Excellent! Thank you!
Merging! :rocket:

**Pull request merged**